rpa-recorder/6.5/modules/ROOT/pages/user-interface.adoc

```diff
@@ -57,16 +57,18 @@ To check your display settings before recording, go to *Windows Settings > Displ
 To capture image-based mouse clicks:
 
 . Start a normal recording process.
-. While recording, press *F3* to start recording image-based mouse clicks, or press the *Image-Based Mouse Clicks* icon on the Recording Toolbar.
-. Capture an image by selecting it on the screen, accept or retake the image, and then set a mouse click, which the Recorder marks with a red dot. Record additional mouse clicks, such as double-mouse clicks, if desired.
+. While recording, press *F3* to start recording image-based mouse clicks, or press the *Image-Based Mouse Clicks* icon on the Recording Toolbar. You can set the initial screen freeze delay from the toolbar by clicking the arrow next to the *Image-Based Mouse Clicks* icon.
+. Capture an image by selecting it on the screen, and then accept or retake the image with or without setting a capture delay. Then, set a mouse click, which the Recorder marks with a red dot. Record additional mouse clicks, such as double-mouse clicks, if desired.
 . Click *Finish*. This generates all relevant data for RPA Builder to create the _Search Pattern_ and _Mouse Action_ Action Steps for the recording.
 
 The process documentation and PDF generated by the Recorder include the recording data. Once you finish using image-based mouse clicks, you can continue with the normal recording process.
 
 When recording image-based mouse clicks in a Web Task:
 
 * Scrolling or zooming in the displayed web page or moving or resizing the browser window results in inaccurate relative clicks in the recording.
-* Navigating to a different web page when capturing Image-Based Mouse Clicks, such as by clicking a link, can cause inaccuracies in the recording of the subsequent relative mouse clicks.
+* Navigating to a different web page when capturing image-based mouse clicks, such as by clicking a link, can cause inaccuracies in the recording of the subsequent relative mouse clicks.
+* The attribute selection panel cannot be opened. Once image-based mouse clicks recording is finished, the panel can be opened.
```

Comment:
note: The panel becomes disabled and the user will not be able to collapse or expand it.